packages/ui-components/src/icons.tsx

```diff
@@ -68,4 +68,29 @@ export class IconUtil {
   static encode(icon: LabIcon): string {
     return 'data:image/svg+xml;utf8,' + encodeURIComponent(icon.svgstr);
   }
+
+  static colorize(
+    icon: LabIcon,
+    fillColor: string,
+    strokeColor?: string
+  ): LabIcon {
+    let svgstr = icon.svgstr;
+
+    if (fillColor) {
```

Comment:
`fillColor` is not listed as an optional param, so either this if is unnecessary or you should add the `?` to the param

Reply:
it has been made optional